SDK_dotnet/Clients/GluwaClient.cs

```diff
@@ -276,10 +308,17 @@ public GluwaClient(Environment env)
             {
                 throw new ArgumentNullException(nameof(target));
             }
-            else if (string.IsNullOrWhiteSpace(getContractAddress(currency)))
+            else if (string.IsNullOrWhiteSpace(currency.ToString()))
             {
                 throw new ArgumentNullException(nameof(currency));
             }
+            else if (paymentID != null)
```

Comment:
use string.IsNullOrWhiteSpace()

Reply:
paymentID is guid..... not string.. can i do this?

Reply:
oh, ya that's fine